src/webview/js/rewriteHtml.js

```diff
@@ -15,10 +15,9 @@ const inlineApiRoutes: RegExp[] = ['^/user_uploads/', '^/thumbnail$', '^/avatar/
  *   2. If the source URL names an endpoint known to require authentication,
  *      inject an API key into its query parameters.
  *
- * DEPRECATED: If no root element is specified, transform every <img> in the
- * entire document.
+ * Private; exported only for tests.
  */
-const rewriteImageUrls = (auth: Auth, element: Element | Document = document) => {
+export const rewriteImageUrls = (auth: Auth, element: Element | Document) => {
```

Comment:
Instead of exporting this, let's just have the tests refer to the renamed `rewriteHTML`.

That way it's entirely an implementation detail of this module that its functionality is broken up into a couple of different functions. That seems like the most appropriate way to think of where its interface is.